### DISCUSSION TITLE
GetUserTopArtists returning forbidden

### DISCUSSION THREAD

**Discussion by TerrizX** [spotify-web-api-java/spotify-web-api-java#276] [Q&A] [answered]
I used the same code on the Get User Top Artists example made here, and yeah. No custom code.

**Comment by dargmuesli** [comment]
Did your access token expire?

**Comment by TerrizX** [comment]
> Did your access token expire?

No

**Comment by dargmuesli** [comment, marked as answer]
Did you use the `user-top-read` scope?

**Reply by TerrizX** [reply]
Defffinetly not lol, thanks I'll see if it works and get back to you if it doesn't
